source/layouts/blog.erb

```diff
@@ -97,12 +100,13 @@
 
       <% if current_article.similar_articles.any? %>
         <section class="blog-related">
-          <p class="blog-related__header">Related</p>
-            <% blog.articles.sample(3).each do |article| %>
+        <p class="blog-related__header">Related</p>
+          <% blog.articles.sample(3).each do |article| %>
+            <% unless article == current_article %>
```

Comment:
Supposedly our [related articles algorithm](https://github.com/ngs/middleman-blog-similar/blob/master/lib/middleman-blog-similar/algorithm.rb#L8) should be handling this. However, if it doesn't seem to work, this guard could be pushed back to the helper itself.